components/utilities/dialog/index.jsx

```diff
@@ -188,23 +193,13 @@ const Dialog = createReactClass({
 	},
 
 	componentDidMount () {
-		if (
-			this.props.position === 'absolute' ||
-			this.props.position === 'relative'
-		) {
+		if (this.props.position === 'absolute' || this.props.position === 'relative') {
 			this.handleOpen();
 		}
 	},
 
 	componentDidUpdate (prevProps, prevState) {
-		if (
-			this.state.triggerPopperJS === true &&
```

Comment:
Ignore these formatting changes, i'll change back. I can't read the code as linted. It hurts my eyes, but I'll change back before committing.